packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -177,9 +179,10 @@ library LibProving {
                     | (uint256(evidence.gasUsed) << 32);
 
                 // Also hash configs that will be used by circuits
-                inputs[9] = uint256(config.blockMaxGasLimit) << 192
-                    | uint256(config.blockMaxTransactions) << 128
-                    | uint256(config.blockMaxTxListBytes) << 64;
+                inputs[9] = uint256(config.blockMaxGasUsed) << 224 // 32 bits
+                    | uint256(config.blockMaxGasLimit) << 192 // 32 bits
+                    | uint256(config.blockMaxTransactions) << 160 // 32 bits
```

Comment:
Which one of the three? `blockMaxTransactions`?

Reply:
yes `blockMaxTransactions `

Reply:
For the next testnet we don't verify any signatures yet so it could be removed. But when we actually do verify signatures in the future we will have a limit, so not sure about the thinking here.